src/Dev/Tasks/CleanupTestDatabasesTask.php

```diff
@@ -2,46 +2,32 @@
 
 namespace SilverStripe\Dev\Tasks;
 
-use SilverStripe\Control\Director;
 use SilverStripe\Dev\BuildTask;
-use SilverStripe\Dev\Deprecation;
+use SilverStripe\PolyExecution\PolyOutput;
 use SilverStripe\ORM\Connect\TempDatabase;
-use SilverStripe\Security\Permission;
-use SilverStripe\Security\Security;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
 
 /**
  * Cleans up leftover databases from aborted test executions (starting with ss_tmpdb)
  * Task is restricted to users with administrator rights or running through CLI.
  */
 class CleanupTestDatabasesTask extends BuildTask
 {
+    protected static string $commandName = 'CleanupTestDatabasesTask';
 
-    private static $segment = 'CleanupTestDatabasesTask';
+    protected string $title = 'Deletes all temporary test databases';
 
-    protected $title = 'Deletes all temporary test databases';
+    protected static string $description = 'Cleans up leftover databases from aborted test executions (starting with ss_tmpdb)';
 
-    protected $description = 'Cleans up leftover databases from aborted test executions (starting with ss_tmpdb)';
+    private static string|array|null $permissions_for_browser_execution = [
+        'anyone_with_dev_admin_permissions' => null,
+        'anyone_with_task_permissions' => null,
+    ];
```

Comment:
```suggestion
    public static function canRunInBrowser(): bool
    {
        if (!parent::canRunInBrowser()) {
            return false;
        }
        return Permission::check('ADMIN');
    }
```
This is a much cleaner way to do the 'must be admin' check rather than overriding the default 'helpful' keys

Reply:
This makes it not configurable.
The intention is that it's configurable.

Reply:
Fair enough, though I think this is a really clunky and un-intuitive way handle permissions. I didn't really grasp what this was for and I was spending a bunch of time peer reviewing this code. The average developer who only glances at this is most likely to not understand what's going on

Instead, perhaps just add a `private static $admin_only_for_browser_execution = false;` on the base class and just set it to true on this job and have that override 'permissions_for_browser_execution'

Reply:
> Is there anywhere else in the core code base we have this pattern of using keys for permissions so they can be toggled off?

Dunno, but we do use this pattern for other things that aren't permissions.

At the end of the day this is the pre-existing pattern that the permissions for these exact parts of the code were already using. I've just moved it, it already existed. I don't want to change it as part of this card.

Reply:
Yeah, the `anyone_with_dev_admin_permissions` and `anyone_with_task_permissions` keys aren't existing though

Does this work? If not would it be feasible to make this work?

'ADMIN',
'ALL_DEV_ADMIN' => true,
'BUILDTASK_CAN_RUN' => true,

Then set to false in this particular job?

Reply:
Yeah that's more intuitive, I can make that work. As discussed in person the top one is preferred so I'll run with that.

Reply:
Done